transactor/src/main/scala/io/mediachain/transactor/Dummies.scala

```diff
@@ -12,6 +13,10 @@ object Dummies {
     }
     override def hashCode = num
     override def toString = "dummy@" + num
+
+    val CBORType = "dummyReference"
```

Comment:
I think we'll have to do some more work to get links encoded properly once the tag is reserved.  

[The spec](https://github.com/ipfs/specs/tree/master/ipld#serialised-cbor-with-tags) basically says if there are no properties attached to the `{"@link": "..."}` object apart from the link itself, then you just write out the link value wrapped in the tag.  

But if you do have properties, you write out an array whose first item is the tagged link and whose second item is a map with the remaining properties.

So the complexity sort of depends on whether we want to support properties in the `@link` object.  Either way, we probably don't want to actually write out the `type` field... But this seemed like a decent way to go for now.